### PR TITLE
Introduced background and handbook organisation, reworked text a bit

### PR DESCRIPTION
**Pull request by kjellp**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [x] Documentation change
- [ ] Language translation

### Pull request long description:
Reworked the front page according to discussions in the handbook working group.

### Changes made:

1) Added brief introduction on what SDA is
2) Reworked the existing text around figure and in table to describe three main archive sub-processes: Submission, Ingestion and Data Retrieval, removed the "Status" column in table
3) Introduced the overall organisation of 4 main section in the text, and briefly their intended content.
4) Kept original "unused" text towards the bottom indicated with a note to be considered reused in section intro pages or elsewhere more suitable. 

### Mentions:

@norling

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,40 +1,68 @@
+
+NeIC Sensitive Data Archive
+===========================
+
+The NeIC Sensitive Data Archive (SDA) software is a modular system of microservices that in alternative configurations can be setup to host sensitive data in an archival service, securely stored encrypted at rest.
```
Comment (norling) [inline docs/index.md:5]: To me, this sentence is a bit complex for a first introduction to the project. Maybe something like:
```suggestion
The NeIC Sensitive Data Archive (SDA) is an encrypted data archive, originally implemented for storage of sensitive biological data. It is implemented as a modular microservice system that can be deployed in different configurations depending on the service needs.
```
would be easier to read for new users?

File: docs/index.md [hunk 2]
```diff
@@ -1,40 +1,68 @@
...
+-   Ingestion - the process of verifying uploaded data and securely storing it in archive storage, while synchronizing state and identifier information with CEGA
+-   Data Retrieval - the process of re-encrypting and staging data for retrieval/download.
 
--   submission - used in the process on submitting and ingesting data.
--   data retrieval - used for data retrieval/download.
 
 
+Service/component | Description | Archive sub-process 
+-------:|:------------|:-----------------------------
+db | A Postgres database with appropriate schema, stores the file header the accession id, file path and checksums as well as other relevant information. | Submission, Ingestion and Data Retrieval
```
Comment (norling) [inline docs/index.md:34]:
```suggestion
db | A Postgres database with appropriate schema, stores the file header, the accession id, file path and checksums as well as other relevant information. | Submission, Ingestion and Data Retrieval 
```

File: docs/index.md [hunk 3]
```diff
@@ -1,40 +1,68 @@
...
--   submission - used in the process on submitting and ingesting data.
--   data retrieval - used for data retrieval/download.
 
 
+Service/component | Description | Archive sub-process 
+-------:|:------------|:-----------------------------
+db | A Postgres database with appropriate schema, stores the file header the accession id, file path and checksums as well as other relevant information. | Submission, Ingestion and Data Retrieval 
+mq (broker) | A RabbitMQ message broker with appropriate accounts, exchanges, queues and bindings. We use a federated queue to get messages from CentralEGA's broker and shovels to send answers back.| Submission and Ingestion 
+Inbox | Upload service for incoming data, acting as a dropbox. Uses credentials from Central EGA. | Submission 
+Intercept | relays message between the queue provided from the federated service and local queues. | Submission and Ingestion
```
Comment (norling) [inline docs/index.md:37]:
```suggestion
Intercept | Relays messages between the queue provided from the federated service and local queues. | Submission and Ingestion 
```

File: docs/index.md [hunk 4]
```diff
@@ -1,40 +1,68 @@
...
--   data retrieval - used for data retrieval/download.
 
 
+Service/component | Description | Archive sub-process 
+-------:|:------------|:-----------------------------
+db | A Postgres database with appropriate schema, stores the file header the accession id, file path and checksums as well as other relevant information. | Submission, Ingestion and Data Retrieval 
+mq (broker) | A RabbitMQ message broker with appropriate accounts, exchanges, queues and bindings. We use a federated queue to get messages from CentralEGA's broker and shovels to send answers back.| Submission and Ingestion 
+Inbox | Upload service for incoming data, acting as a dropbox. Uses credentials from Central EGA. | Submission 
+Intercept | relays message between the queue provided from the federated service and local queues. | Submission and Ingestion 
+[Ingest](services/ingest.md) | Splits the Crypt4GH header and moves it to database. The remainder of the file is sent to the storage backend (archive). No cryptographic tasks are done. | Ingestion
```
Comment (norling) [inline docs/index.md:38]:
```suggestion
[Ingest](services/ingest.md) | Splits the Crypt4GH header and moves it to the database. The remainder of the file is sent to the storage backend (archive). No cryptographic tasks are done. | Ingestion 
```

File: docs/index.md [hunk 5]
```diff
@@ -1,40 +1,68 @@
...
 
 
+Service/component | Description | Archive sub-process 
+-------:|:------------|:-----------------------------
+db | A Postgres database with appropriate schema, stores the file header the accession id, file path and checksums as well as other relevant information. | Submission, Ingestion and Data Retrieval 
+mq (broker) | A RabbitMQ message broker with appropriate accounts, exchanges, queues and bindings. We use a federated queue to get messages from CentralEGA's broker and shovels to send answers back.| Submission and Ingestion 
+Inbox | Upload service for incoming data, acting as a dropbox. Uses credentials from Central EGA. | Submission 
+Intercept | relays message between the queue provided from the federated service and local queues. | Submission and Ingestion 
+[Ingest](services/ingest.md) | Splits the Crypt4GH header and moves it to database. The remainder of the file is sent to the storage backend (archive). No cryptographic tasks are done. | Ingestion 
+[Verify](services/verify.md) | Uses a crypt4gh secret key, this service can decrypt the stored files and checksum them against the embedded checksum for the unencrypted file. | Ingestion
```
Comment (norling) [inline docs/index.md:39]:
```suggestion
[Verify](services/verify.md) | Using the archive crypt4gh secret key, this service can decrypt the stored files and checksum them against the embedded checksum for the unencrypted file. | Ingestion 
```

**Review by norling — APPROVED** [review]
Neat! I have some nitpicky grammary comments, but feel free to ignore them if you don't think they apply! :)

**Comment by kjellp** [comment]
All very good suggestions Martin, accepted and commited now:)

**Comment by blankdots** [comment]
please add the spelling words highlighted in https://github.com/neicnordic/neic-sda/actions/runs/3743236980/jobs/6355162208#step:4:1 i `docs/dictionary/wordlist.txt` so that next time they are ignored

**Comment by kjellp** [comment]
@blankdots Thanks for the instructions, words added now :)

**Review by blankdots — APPROVED** [review]

**Review by joshbaskaran — APPROVED** [review]
LGTM

**Pull request merged**